build.sbt

```diff
@@ -162,29 +166,34 @@ lazy val sesame = Project("sesame", file("sesame"), settings = commonSettings)
     )
   ).dependsOn(rdfJVM, ntriplesJVM, rdfTestSuiteJVM % "test->compile")
 
-lazy val jsonldJS = Project("jsonld", file("jsonld.js"), settings = commonSettings)
+lazy val jsonldJS = Project("jsonld", file("jsonld.js"))
+  .settings(commonSettings: _*)
+  .settings()
```

Comment:
is this empty settings doing any work?

Reply:
no, this is a mistake

Reply:
ok, should be fixed then for the PR to go through :-)